src/core/services/actions-handler/actions-handler.js

```diff
@@ -114,6 +114,12 @@ export default class ActionsHandler extends LitElement {
 
     this.addEventListener('adminAccess', ({ detail }) => {
       const { category, action } = detail.event;
+
+      // keep existing params and add new one
+      const url = new URL(window.location.href);
+      url.searchParams.append('admin', 1);
+      window.location.search = url.search;
+
       this.loanAnanlytics?.sendEvent(category, action, this.identifier);
```

Comment:
This should happen before we change the location ; otherwise it might not get hit!